InvoiceMaker/InvoiceMaker/FormModels/CreateInvoice.cs

```diff
@@ -0,0 +1,36 @@
+﻿using InvoiceMaker.Data;
+using InvoiceMaker.Models;
+using InvoiceMaker.Models.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace InvoiceMaker.FormModels
+{
+    public class CreateInvoice
+    {
+        public IList<Client> Clients { get; set; }
+        public int ClientId { get; set; }
+        public string Name { get; set; }
+        public string InvoiceNumber { get; set; }
+        public InvoiceStatus Status { get; set; }
+
+        private Context _context = new Context();
```

Comment:
A form model class should never instantiate an instance of the database context. If this class needs a context instance, you should pass the instance from the controller into this class.